go-controller/pkg/metrics/ovn.go

```diff
@@ -279,6 +306,30 @@ func ovnControllerConfigurationMetricsUpdater() {
 	}
 }
 
+func ovnControllerSubnetMetricsUpdater() {
+	var serviceSubnets string
+	var clusterSubnets string
+
+	for index, cidr := range config.Kubernetes.ServiceCIDRs {
+		if index == 0 {
+			serviceSubnets = serviceSubnets + cidr.String()
+		} else {
+			serviceSubnets = serviceSubnets + "," + cidr.String()
+		}
+	}
+
+	for index, subnet := range config.Default.ClusterSubnets {
+		if index == 0 {
```

Comment:
If we decide to split the labels per IP protocol I think we should use `utilnet.IsIPv6CIDR` to assign the subnet into its correct "bucket". Relying on the index seems incorrect since there is no restriction imposed on the order at which these IPv4/IPv6 subnets are passed to ovn-kubernetes.